src/v/cluster/partition_probe.cc

```diff
@@ -59,6 +59,27 @@ void replicated_partition_probe::setup_internal_metrics(const model::ntp& ntp) {
       partition_label(ntp.tp.partition()),
     };
 
+    // The following few metrics uses a separate add_group call which doesn't
+    // aggregate any labels since aggregation does not make sense for "leader
+    // ID" values.
+    _metrics.add_group(
+      prometheus_sanitize::metrics_name("cluster:partition"),
```

Comment:
Maybe make "cluster:partition" a constant/variable.

Reply:
Made it a namespace-scope const global.